app/views/search/_granule_search.html.erb

```diff
@@ -78,7 +78,7 @@
           <span data-bind="visible: !cmrGranuleQuery.validateCloudCoverRange(cmrGranuleQuery.cloudCover.min(),cmrGranuleQuery.cloudCover.max())" class="error">Minimum must be less than Maximum</span>
         </p>
       </dd>
-      <!-- ko if: $root.labs() && searchable_attributes -->
+      <!-- ko if: $root.labs() && searchable_attributes || tags() && tags()['<%=Rails.configuration.cmr_tag_namespace.gsub(/\.$/, '')%>.features.attribute_search'] && tags()['<%=Rails.configuration.cmr_tag_namespace.gsub(/\.$/, '')%>.features.attribute_search'].data -->
```

Comment:
I will edit the PR applying the later two comments. About the first one: Doug sees tags like
```            {"concept-id": "T1230649091-CMR",
            "revision-id": 1,
            "tag-key": "edsc..extra.browseable_granule",
            "description": null,
            "originator-id": "edsc.sys"}```
The pre-configured value ```edsc``` doesn't have a trailing dot though.